src/libs/ajax/workflows-app/Cbas.js

```diff
@@ -7,6 +7,10 @@ export const Cbas = (signal) => ({
     const res = await fetchFromProxy(cbasUrlRoot)('status', _.mergeAll([authOpts(), { signal, method: 'GET' }]));
     return res.json();
   },
+  info: async (cbasUrlRoot) => {
```

Comment:
I'm going to suggest we hold off on a contract test for this to avoid dragging this PR out. I'm not 100% sure how CBAS will verify this, given that this endpoint doesn't go through the controller layer

Reply:
I agree with this as long as there's an existing ticket so we don't forget.

Reply:
Also, do you mind explaining why we need the build info to archive?